src/runtime/config/configuration-qemu-nvidia-gpu-snp.toml.in

```diff
@@ -698,3 +698,7 @@ create_container_timeout = @DEFAULTTIMEOUT_NV@
 # to the hypervisor.
 # (default: /run/kata-containers/dans)
 dan_conf = "@DEFDANCONF@"
+
+# Enforce guest pull. This instructs the runtime to communicate to the agent via annotations that
+# the container image should be pulled in the guest, without using an external snapshotter.
+# force_guest_pull = true
```

Comment:
Would you want to follow the established practice of having a `@DEFFORCEGUESTPULL@` set from the makefile?

Asking because I suspect we will soon want different defaults for different configs on that one. (Something like `@DEFFORCEGUESTPULLTEE@` vs. `@DEFFORCEGUESTPULL@` ?)

Reply:
done